adoc/deployment-aws.adoc

```diff
@@ -0,0 +1,87 @@
+== Deployment on Amazon AWS
+
+.Preparation Required
+[NOTE]
+You must have completed <<deployment.preparations>> to proceed.
+
+You will use {tf} to deploy the required master and worker cluster nodes (plus a load
+balancer) to {soc} and then use the `skuba` tool to bootstrap the {kube} cluster on
+top of those.
+
+. Download the AWS credentials
+.. Log in to the AWS console.
+.. Click on your username in the upper right hand corner to reveal the dropdown menu.
+.. Click on My Security Credentials.
+.. Click Create Access Key on the Security Credentials tab.
+.. Note down the newly created _Access_ and _Secret_ keys.
+
+=== Deploying the cluster nodes
+
+. Find the {tf} template files for {soc} in `/usr/share/caasp/terraform/aws`.
```

Comment:
Again `{soc}` the referred filename is `/aws`

Reply:
Fixed.